src/server/api/routes/api-router.js

```diff
@@ -53,6 +55,7 @@ router.use('/allUsers', allUsersRouter);
 router.use('/qr', qrRouter);
 
 router.use('/game-instances', gameInstancesRouter);
-router.use('/newGame', newGameRouter);
+router.use('/new-game', createNewGameRouter);
+router.use('/game_factories', getQuestionByIdRouter);
```

Comment:
Could you rename this one as well?

```suggestion
router.use('/game-factories', getQuestionByIdRouter);
```

Reply:
yes i can, but I thing it was already changed in another PR I think @Lucha2040  did that.. coz this related to another one not mine